scripts/image-version.sh

```diff
@@ -20,28 +24,36 @@ unpinImageVersion(){
   sed "/^${IMG_VAR}=/d" scripts/image_versions.src
 }
 pinnedImages(){
-  for PREFIX in ${VAR_PREFIXES_ARR[@]}; do
+  for PREFIX in "${VAR_PREFIXES_ARR[@]}"; do
```

Comment:
Undo this change -- `${VAR_PREFIXES_ARR[@]}` should be treated as separate arguments.